README.md

```diff
@@ -48,7 +48,7 @@ jobs:
       - uses: actions/checkout@v3
 
       - name: Start LocalStack
-        uses: HarshCasper/setup-localstack@v0.1.2
+        uses: LocalStack/setup-localstack@v0.1.2
```

Comment:
nit: could be updated to `v0.1.3`, for consistency with the version above
```suggestion
        uses: LocalStack/setup-localstack@v0.1.3
```

Reply:
In general, how do you feel about introducing a `latest` tag for this action @HarshCasper ? It would require a bit of additional effort every time we release a new version (tagging the latest release commit with the git tag `latest`), but then users could simply refer to this in their pipelines:

```
        uses: LocalStack/setup-localstack@latest
```

Might come in handy, especially if we anticipate that there will be more updates coming for this Action in the near future.. 👍

Reply:
Let me see how to set it up 👍 

I will do this in a follow-up PR